test/integration/job/job_test.go

```diff
@@ -1174,6 +1174,227 @@ func TestBackoffLimitPerIndex(t *testing.T) {
 	}
 }
 
```

Comment:
Wouldn't we need to add integration test for the enabling and disabling then enabling JobManagedBy featureGate?

Reply:
Makes sense, added the `TestManagedBy_Reenabling` test